server/monitor_test.go

```diff
@@ -4212,13 +4212,29 @@ func TestMonitorJsz(t *testing.T) {
 				t.Fatalf("expected single account")
 			}
 			acc := info.AccountDetails[0]
-			got := acc.ReservedMemory
-			expected := uint64(5242880)
```

Comment:
I would have kept expected as uint64() since after all acc.ReservedMemory is a uint64.